tests/python-e2e/ops.py

```diff
@@ -25,6 +27,67 @@ def test_read_write_family(self):
             read_text = rw_file.readline()
             self.assertEqual(read_text, TEXT)
 
+    def test_fopen(self):
+        """
+        Opens a file in "/tmp/"
+        """
+        file_path, _ = self._create_new_tmp_file()
+        self.assertFalse(self._check_path_exists_on_host(file_path))
+        libc = CDLL('libc.so.6')
+        fp = libc.fopen(file_path.encode(), 'rb')
+        str_len = len(TEXT)
+        buff = (c_ubyte * len(TEXT))() # Creates empty array of bytes
+        libc.fread(buff, len(TEXT), 1, fp)
+        txt = "".join([chr(i) for i in buff])
+        self.assertEqual(txt, TEXT)
+
+    def test_fdopen(self):
+        """
+        Opens a file in "/tmp"
+        """
+        file_path, _ = self._create_new_tmp_file()
+        self.assertFalse(self._check_path_exists_on_host(file_path))
+        libc = CDLL('libc.so.6')
+        fp = libc.fopen(file_path.encode(), 'rb')
+        fdesc = libc.fileno(fp)
+        fp2 = libc.fdopen(fdesc, "r")
+        str_len = len(TEXT)
+        buff = (c_ubyte * len(TEXT))() # Creates empty array of bytes
+        libc.fread(buff, len(TEXT), 1, fp2)
+        txt = "".join([chr(i) for i in buff])
+        self.assertEqual(fd, 3)
```

Comment:
I think the best flow should be to call fdopen, then read and just assert the text

Reply:
Done